src/aks-preview/azext_aks_preview/managed_cluster_decorator.py

```diff
@@ -3128,6 +3095,39 @@ def update_load_balancer_profile(self, mc: ManagedCluster) -> ManagedCluster:
             )
         return mc
 
+    def update_outbound_type_in_network_profile(self, mc: ManagedCluster) -> ManagedCluster:
+        """Update outbound type of network profile for the ManagedCluster object.
+        :return: the ManagedCluster object
+        """
+        self._ensure_mc(mc)
+
+        outboundType = self.context.get_outbound_type()
+        if outboundType:
+            mc.network_profile.outbound_type = outboundType
+        return mc
+
+    def update_nat_gateway_profile(self, mc: ManagedCluster) -> ManagedCluster:
```

Comment:
If the implementation here is exactly the same as in azure-cli, may omit the func

Reply:
removed. Thanks!